src/test/ceph-helpers.sh

```diff
@@ -21,6 +21,11 @@ CEPH_HELPER_VERBOSE=false
 TIMEOUT=60
 PG_NUM=4
 
+XMLSTARLET=xmlstarlet
+if lsb_release -si | grep -qi suse; then
```

Comment:
I've learnt the hard way that lsb_release is often not available by default, nor is the **which** command. Maybe using **type xmlstartlet** and fallback to **xml** is more reliable ?